### PR TITLE
core: use LCP specific message for NO_LCP

### LINKED ISSUE #14554: Make NO_LCP error message specific to LCP
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://jsbin.com/pajibab/edit?html,output

### What happened?

When analyzing page load with mode: navigation, device: mobile and categories: performance there is an error on the LCP metric. 
We receive the following message:
`Something went wrong with recording the trace over your page load. Please run Lighthouse again. (NO_LCP)`

This error error does not occur when analyzing page load desktop on desktop.

### What did you expect?

Would have expected a similar result as when running the test on desktop. Where the error does not occur.

### What have you tried?

Made a minimal html page with only the image with css style object-fit: cover defined to make sure nothing else might be causing the issue.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

9.6.6

### Chrome Version

107.0.5304.110

### Node Version

_No response_

### OS

Mac 

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by adamraine**
Closes #14554

### CODE REVIEW

File: core/lib/lh-error.js [hunk 1]
```diff
@@ -23,6 +23,11 @@ const UIStrings = {
    * @example {NO_FCP} errorCode
    * */
   noFcp: 'The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. ({errorCode})',
+  /**
+   * @description Error message explaining that the Largest Contentful Paint metric was not seen during the page load.
+   * @example {NO_LCP} errorCode
+   * */
+  noLcp: 'The page did not paint anything that qualifies as LCP. Ensure the page has a valid LCP element and then try again. ({errorCode})',
```
Comment (brendankenny) [inline core/lib/lh-error.js:30]: possible tweaks:

```suggestion
  noLcp: 'The page did not display content that qualifies as a Largest Contentful Paint (LCP). Ensure the page has a valid LCP element and then try again. ({errorCode})',
```

Comment (brendankenny) [reply core/lib/lh-error.js:30]: should we just delete the `errorCode` part now that it's specific?

**Review by brendankenny — COMMENTED** [review]
nice, much better

File: core/lib/lh-error.js (hunk 1 above)
Comment (adamraine) [reply core/lib/lh-error.js:30]: > should we just delete the errorCode part now that it's specific?

This message is also used for `NO_LCP_ALL_FRAMES`.

Comment (brendankenny) [reply core/lib/lh-error.js]: OTOH we have [pretty good SEO on "NO_LCP", apparently](https://www.google.com/search?q=NO_LCP), so we should probably just leave it :)

**Review by brendankenny — APPROVED** [review]

**Pull request merged**